multimodal/src/autogluon/multimodal/data/infer_types.py

```diff
@@ -250,12 +256,19 @@ def infer_column_types(
         elif is_numerical_column(data[col_name], valid_data[col_name]):  # Infer numerical column
             column_types[col_name] = NUMERICAL
         elif is_imagepath_column(data[col_name], col_name):  # Infer image-path column
-            column_types[col_name] = IMAGE_PATH
+            column_types[col_name] = IMAGE
```

Comment:
What is the difference between IMAGE and IMAGE_PATH?

Reply:
They are the same currently, meaning image type. Since we use `image` in the `data_types` and we want to it in the filtering, it's better to change to `image` here. We still keep the `image_path` in the dataframe preprocessor for backward compatibility.